donation-ticker.html

```diff
@@ -46,7 +46,7 @@ <h1 class="h5">{{ entity.sumDonations | formatMoney }} / {{ entity.fundraisingGo
 		Vue.filter(
 			'formatDate',
 			(value) => {
-				return moment(value).format('MM/DD/YYYY hh:mm');
+				return dayjs(value).format('MM/DD/YYYY hh:mm a');
```

Comment:
@reverk bonus points if you put the date mask into `window.config.donationTicker` inside [config.js](https://github.com/DonorDrive/StreamKit/blob/master/config.js)